### ISSUE TITLE
Appium is Unable to Find iOS Accessibility Identifiers

### ISSUE THREAD

**Issue by T61Z8T3** [appium/appium-inspector#229]
## Appium or Appium Desktop?

- [ ☑️ ] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

It's about a few days that Appium is unable to find the iOS accessibility IDs on some of the UITableView. The locators could be found before but by new version of our app we found this issue. I discussed it with our developers, they set view.accessibilityIdentifier = @"aName" and view.isAccessibilityElement = YES, but still these accessibility IDs can not be seen in Appium.
We had some new features in our new version of App but Appium is Unable to Find iOS Accessibility Identifiers on the screens which had no changes at all.

This is an screenshot of our previous version of app: 
![image](https://user-images.githubusercontent.com/78351604/145416420-d8305c2c-945b-4577-9553-329989694ad1.png)

This is an screenshot of our new version of app: 
![image](https://user-images.githubusercontent.com/78351604/145416511-2b14d7a5-f38c-4acd-9837-4cd5be33bcf7.png)

## Environment

- I am running Appium Desktop version <1.22.1>.
- I am on:
    -  Mac version 11.6.1 (20G224)

**Comment by mykola-mokhnach** [comment]
I have doubts this issue has anything to do with Appium. Does the app UI look different in the Xcode accessibility inspector?

**Comment by T61Z8T3** [comment]
> I have doubts this issue has anything to do with Appium. Does the app UI look different in the Xcode accessibility inspector?

You are right, I think it was something from development part. We just found out the reason. Thank you for your answer.

**Comment by mykola-mokhnach** [comment]
We'd appreciate it if you could also share more details about the applied solution. It might be useful to other people having a similar development stack.

**Comment by T61Z8T3** [comment]
The elements can be found in Xcode accessibility inspector. So it was about defining the elements.
